lattice-control/README.md

```diff
@@ -1,12 +1,60 @@
-![Crates.io](https://img.shields.io/crates/v/wasmcloud-interface-lattice-control)
-[![Documentation](https://img.shields.io/badge/Docs-Documentation-blue)](https://wasmcloud.dev)
-[![Rustdocs](https://docs.rs/wasmcloud-interface-lattice-control/badge.svg)](https://docs.rs/wasmcloud-interface-lattice-control)
-
-# Lattice Control Interface
+[![crates.io](https://img.shields.io/crates/v/wasmcloud-interface-lattice-control.svg)](https://crates.io/crates/wasmcloud-interface-lattice-control)&nbsp;
+<!-- [![TinyGo Version](https://img.shields.io/github/go-mod/go-version/wasmcloud/interfaces?label=TinyGo&filename=lattice-control%2Ftinygo%2Fgo.mod)](https://pkg.go.dev/github.com/wasmcloud/interfaces/lattice-control/tinygo) -->
```

Comment:
nit. we should probably just delete this instead of HTML commenting them.

Reply:
removed 👍🏻